fastestimator/pipeline.py

```diff
@@ -401,6 +403,8 @@ def __call__(self,
                  epoch: int = 1,
                  ds_id: Optional[str] = None,
                  shuffle: Optional[bool] = None,
+                 train_steps_per_epoch: Optional[int] = None,
```

Comment:
doesn't make sense to have 'train_steps_per_epoch' and 'eval_steps_per_epoch' here since this method is specific to a given mode already. Maybe just a single arg 'steps' and have the estimator decide which argument to pass given the context